src/components/main/ItemScreen.js

```diff
@@ -75,7 +77,10 @@ const ItemScreen = () => {
 
   return (
     <Main>
-      <ItemMain item={item}>{content}</ItemMain>
+      <UppyContextProvider enable={enableEdition} itemId={itemId}>
```

Comment:
Not specifically for this PR, but in English, this should be `enableEditing`.

Reply:
It's not too much changes, I've applied it 👍